extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -166,3 +166,61 @@
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+/** Inline attachment specifications */
+.i-amphtml-story-inline-page-attachment-chip {
+  display: flex !important;
+  position: relative !important;
+  border-radius: 24px !important;
+  background-color: rgba(255, 255, 255, 0.5) !important;
+  color: black !important;
+  align-items: center !important;
+  overflow: hidden !important;
+  margin-bottom: 12px !important;
+}
+
+.i-amphtml-story-inline-page-attachment-chip:only-child {
+  margin-bottom: 20px !important;
+}
+
+.i-amphtml-story-inline-page-attachment-img,
+.i-amphtml-story-inline-page-attachment-img-2,
+.i-amphtml-story-inline-page-attachment-arrow {
+  vertical-align: middle !important;
+  background-size: contain !important;
+  background-repeat: no-repeat !important;
+}
+
+.i-amphtml-story-inline-page-attachment-img,
+.i-amphtml-story-inline-page-attachment-img-2 {
+  height: 48px !important;
+  width: 48px !important;
+  background-color: white !important;
+}
+
+.i-amphtml-story-inline-page-attachment-img-2 {
+  display: none !important;
+  margin-left: 1px !important;
```

Comment:
If building these dynamically (comment in amp-story-open-page-attachment.js),
these can have the same class  `i-amphtml-story-inline-page-attachment-img` and the second child can be targeted using `.i-amphtml-story-inline-page-attachment-img:nth-child(2)`.
This way the `i-amphtml-story-inline-page-attachment-img-2` class can be removed.